.all-contributorsrc

```diff
@@ -1162,6 +1162,15 @@
       "contributions": [
         "doc"
       ]
+    },
+       {
+      "login": "AreloTanoh",
+      "name": "Arelo Tanoh",
+      "avatar_url": " ",
```

Comment:
that's the link to your user picture from your account 
```suggestion
      "avatar_url": "https://avatars.githubusercontent.com/u/87912196?v=4",
```

Reply:
Ah i see. How do i leave that blank ?

Reply:
@mloning made some changes so hope will work